Tests/CoreTests/Sources/Helper/VersionTest.swift

```diff
@@ -0,0 +1,10 @@
+import XCTest
+@testable import AblyAssetTrackingCore
+
+class VersionTest: XCTestCase {
+    private let libraryVersion = "1.0.0-beta.3"
+    
+    func testCurrentVersionIsSetAndSemantic() {
+        XCTAssertEqual(libraryVersion, Version.libraryVersion)
+    }
+}
```

Comment:
I'm not sure if this test is bringing any value to the project :thinking: I mean that you are just comparing two hardcoded values. And if the version changes then you will need to change the hardcoded value here too, right? Maybe instead of that, the test could check if the library version is compliant with the SEMVER standard? They have an [official regex](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) that you could use to check that :wink: WDYT?

Reply:
I'm resolving this as this was already mentioned and discussed in [a previous comments](https://github.com/ably/ably-asset-tracking-swift/pull/186#issuecomment-921819602).